From bf916aafc3df7bc1f6bc4da55a59f4876a1ed647 Mon Sep 17 00:00:00 2001
From: Brad Geltz <brad.geltz@intel.com>
Date: Wed, 5 Aug 2020 13:34:15 -0700
Subject: [PATCH 3/8] Link w/GEOPM
#
#  Copyright (c) 2015 - 2023, Intel Corporation
#  SPDX-License-Identifier: BSD-3-Clause
#

- Also use default compilers (not Intel wrappers)

Signed-off-by: Brad Geltz <brad.geltz@intel.com>
---
 test/example1/makenek-intel | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/test/example1/makenek-intel b/test/example1/makenek-intel
index a5cce80..82ec388 100755
--- a/test/example1/makenek-intel
+++ b/test/example1/makenek-intel
@@ -6,10 +6,10 @@
 SOURCE_ROOT="../../src"
 
 # Fortran compiler
-F77="mpiifort"
+F77="${MPIF77} ${GEOPM_FFLAGS}"
 
 # C compiler
-CC="mpiicc"
+CC="${MPICC}"
 
 # pre-processor symbol list
 # (set PPLIST=? to get a list of available symbols)
@@ -29,10 +29,10 @@ PPLIST="TIMERS CGTIMERS"
 #USR="foo.o"
 
 # linking flags
-USR_LFLAGS="-qopenmp -mcmodel=medium -shared-intel"
+USR_LFLAGS="-qopenmp -mcmodel=medium -shared-intel ${GEOPM_LDFLAGS} ${GEOPM_LDLIBS} ${GEOPM_FORTRAN_LDLIBS}"
 
 # generic compiler flags
-#G="-g"
+G="-DGEOPM -I${GEOPM_ROOT}/include"
 
 # optimization flags
 OPT_FLAGS_STD="-qopenmp -O3 -g -xCORE-AVX2 -mcmodel=medium -shared-intel"
-- 
2.23.0

